### DISCUSSION TITLE
Quick Start issue

### DISCUSSION THREAD

**Discussion by GervaisdeM-SpryPoint** [cloudposse/community#22] [Essential Support] [answered]
Hi folks,

The company I work with is using the Quick start.

I'm just starting out with the very early steps for set up and when I run the `atmos workflow init/tfstate -f baseline` command, I am getting errors.

First, I get an eks error that you can see here when I run a validate:

```
 √ : [superadmin] (HOST) spryops-infrastructure ⨠ atmos validate stacks
no matches found for the import 'catalog/eks/clusters/default' in the file 'catalog/eks/clusters/auto.yaml'
Error: failed to find a match for the import '/localhost/code/spryops-infrastructure/stacks/catalog/eks/clusters/default.yaml' ('/localhost/code/spryops-infrastructure/stacks/catalog/eks/clusters' + 'default.yaml')
```

I commented out the reference to default in the auto.yaml and then I get another error:

```
 √ : [superadmin] (HOST) spryops-infrastructure ⨠ atmos validate stacks
no matches found for the import 'catalog/iam-service-linked-roles' in the file 'orgs/spt/core/auto/global-region/github.yaml'
Error: failed to find a match for the import '/localhost/code/spryops-infrastructure/stacks/catalog/iam-service-linked-roles.yaml' ('/localhost/code/spryops-infrastructure/stacks/catalog' + 'iam-service-linked-roles.yaml')
```

Can anyone help me out here? Should the `atmos workflow init/tfstate -f baseline` just work or am I expected to create that default file and the iam-service-linked-roles.yaml? If so, can someone point me to some docs on what needs to be in these files?

Thanks!

**Comment by GabisCampana** [comment]
@Benbentwo @milldr

**Reply by osterman** [reply]
@GervaisdeM-SpryPoint: We've identified the problem and will reach out today with the updated configs. We're thankful for your report and apologies for the bug/delays this has caused.

**Comment by Benbentwo** [comment, marked as answer]
So this was caused by an issue with the generation process not fully cleaning up unused files.

I've sent over another zip that should have a couple of improvements.

For clarity however:
The fix is to as you've done remove the stack configs for EKS if you are an ECS engagement.
`orgs/spt/core/auto/global-region/github.yaml`
should look similar to 
```yaml
import:
  - orgs/acme/core/auto/_defaults
  - mixins/region/us-east-1
  - catalog/philips-labs-github-runners
```

for ECS, as `catalog/iam-service-linked-roles` is an EKS component. Similarly, there shouldn't be any EKS stacks, so we can remove that catalog folder and any references to it (search repo for `catalog/eks` and can remove those imports

**Reply by Benbentwo** [reply]
Feel free to reopen if this isn't clear or fix your issue
